docs/docs/configuration/zones.md

```diff
@@ -131,3 +130,17 @@ cameras:
         objects:
           - car
 ```
+
+### Loitering Time
+
+Zones support a `loitering_time` configuration which can be used to only consider an object as part of a zone if they loiter for the specified number of seconds. This can be used, for example, to create alerts for cars that stop on the street but not cars that just drive past your camera.
```

Comment:
```suggestion
Zones support a `loitering_time` configuration which can be used to only consider an object as part of a zone if they loiter in the zone for the specified number of seconds. This can be used, for example, to create alerts for cars that stop on the street but not cars that just drive past your camera.
```